File: app/Shared/TunnelLog.swift
import Foundation
import os

enum TunnelLog {
    private static let logger = Logger(subsystem: HopConstants.mainBundleID, category: "Tunnel")
    private static let prefix = "ɹǝddoH"

    static func info(_ message: String) {
        logger.info("\(message, privacy: .public)")
        NSLog("[\(prefix)] %@", message)
    }

    static func error(_ message: String) {
        logger.error("\(message, privacy: .public)")
        NSLog("[\(prefix) ERROR] %@", message)
    }
}

File: app-android/app/src/main/java/com/aengix/hopper/util/TunnelLog.kt
package com.aengix.hopper.util

import android.util.Log

object TunnelLog {
    private const val TAG = "Hopper"

    fun info(message: String) {
        Log.i(TAG, message)
    }

    fun error(message: String) {
        Log.e(TAG, message)
    }
}
